ui/components/datasetItemWorkspace/src/components/DatasetItemViewer/ImageViewer.svelte

```diff
@@ -27,56 +32,122 @@
     itemObjects,
     preAnnotationIsActive,
     colorScale,
+    filters,
+    itemMetas,
     selectedKeypointsTemplate,
   } from "../../lib/stores/datasetItemWorkspaceStores";
   import { updateExistingObject } from "../../lib/api/objectsApi";
   import { templates } from "../../lib/settings/keyPointsTemplates";
 
+  // Attributes
   export let selectedItem: ImageDatasetItem;
   export let embeddings: Record<string, ort.Tensor>;
   export let currentAnn: InteractiveImageSegmenterOutput | null = null;
-  export let brightness: number;
-  export let contrast: number;
 
-  let imagesPerView: Record<string, HTMLImageElement[]> = {};
+  // Images per view type
+  type ImagesPerView = Record<string, HTMLImageElement[]>;
+  let imagesPerView: ImagesPerView = {};
+  let loaded: boolean = false; // Loading status of images per view
+
+  /**
+   * Normalize the pixel values of an image to a specified range.
+   * @param image The image to normalize.
+   * @param min The minimum pixel value.
+   * @param max The maximum pixel value.
+   */
+  const normalizeRange = (image: ImageJS, min: number, max: number): void => {
+    image.bitDepth = 8;
+    image.maxValue = 255;
+
+    const nPixels: number = image.size;
+    for (let i = 0; i < nPixels; ++i) {
+      let pixel: number = image.data[i];
+      pixel = pixel < min ? 0 : pixel > max ? 255 : ((pixel - min) / (max - min)) * 255;
+      image.data[i] = pixel;
+    }
+  };
+
+  /**
+   * Load images from the given views.
+   * @param views The views to load images from.
+   * @returns A promise that resolves to the loaded images per view.
+   */
+  const loadImages = async (views: Record<string, ItemView>): Promise<ImagesPerView> => {
+    const images: ImagesPerView = {};
+    const promises: Promise<void>[] = Object.entries(views).map(async ([key, value]) => {
+      const img: ImageJS = await ImageJS.load(`/${value.uri}`);
+
+      $itemMetas.color = img.channels === 1 ? "grayscale" : "rgba";
+      $itemMetas.format = (img.bitDepth as number) === 8 ? "8bit" : "16bit";
+
+      if ($itemMetas.format === "16bit" && $itemMetas.color === "grayscale") {
```

Comment:
Is it a requirement to normalize only non 8bit grayscale ? 
- test is not 16bit, it's non 8bit (from [ImageJS doc](https://github.com/image-js/image-js/blob/9ab86a86f6c13a9a7d14c62566c1396c3c6f54f4/src/image/Image.js#L36)) it could be 1, 8 or 16 (maybe 32 also, not clear)
- normalize only grayscale ?

Reply:
I don't know if normalize is the right term for the function, its purpose is to let us convert the image from 16-bit to 8-bit ourselves, rather than use the default HTML/Konva conversion. It also allows us to select the range of pixel values we want to display. We could technically add support for 8-bit images here, but I don't know if that is the right place to do so, as we could also use a custom Konva filter in Canvas2D together with the other filters. About grayscale, I only implemented the support for it because Jaonary told me that the 16-bit images of the user would be grayscale. It is possible however to easily add support for RGB(A) 16-bit images.